app/src/main/java/net/gsantner/opoc/util/GsContextUtils.java

```diff
@@ -2880,8 +2881,19 @@ public boolean isDarkModeEnabled(final Context context) {
 
     public static void blinkView(final View view) {
         if (view != null) {
-            final float init = view.getAlpha();
-            ObjectAnimator.ofFloat(view, View.ALPHA, init, 0.1f, 1.0f, 0.1f, 1.0f, init).setDuration(1000).start();
+            final Object tag = view.getTag(R.id.view_animator);
+            if (tag != null && tag instanceof ObjectAnimator) {
+                if (((ObjectAnimator) tag).isRunning()) {
+                    return;
+                } else {
+                    view.setTag(R.id.view_animator, null);
```

Comment:
I believe this function is only used to blink the view in showFile. 

I am 100% ok with replacing this use with your approach to highlighting the background. 

Either way this is a good change.